app/assets/stylesheets/ghetto_styles.css.scss

```diff
@@ -0,0 +1,1500 @@
+  color: #555;
+}
+/* line 29, ../../app/assets/stylesheets/new_styles/_forms.scss */
+form.block-form fieldset {
+  -moz-border-radius: 5px 5px 5px 5px;
+  -webkit-border-radius: 5px 5px 5px 5px;
+  border-radius: 5px 5px 5px 5px;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.2), 0 1px 2px rgba(255, 255, 255, 0.7);
+  margin-bottom: 1em;
+  background-color: #fff;
+  border: 1px solid #888888;
+  /* mainly bootstrap overrides */
+}
+/* line 42, ../../app/assets/stylesheets/new_styles/_forms.scss */
+form.block-form fieldset input[type=text],
+form.block-form fieldset input[type=email],
+form.block-form fieldset input[type=password] {
+  -webkit-box-shadow: 0 0 0 0;
+  -moz-box-shadow: 0 0 0 0;
+  box-shadow: 0 0 0 0;
+  -moz-border-radius: 0 0 0 0;
+  -webkit-border-radius: 0 0 0 0;
+  border-radius: 0 0 0 0;
+  background: transparent;
+  padding: 10px;
+  margin-bottom: 0;
+  border: none;
+}
+/* line 55, ../../app/assets/stylesheets/new_styles/_forms.scss */
+form.block-form fieldset .control-group {
+  margin: 0;
+  border-bottom: 1px solid #dddddd;
+}
+/* line 62, ../../app/assets/stylesheets/new_styles/_forms.scss */
+form.block-form fieldset .control-group .control-label,
+form.block-form fieldset .control-group input[type=text],
+form.block-form fieldset .control-group input[type=password],
+form.block-form fieldset .control-group .field_with_errors label {
+  padding: 10px;
+  margin: 0;
+}
+/* line 67, ../../app/assets/stylesheets/new_styles/_forms.scss */
+form.block-form fieldset .control-group .controls {
+  margin-left: 100px;
+  position: relative;
+}
+/* line 68, ../../app/assets/stylesheets/new_styles/_forms.scss */
+form.block-form fieldset .control-group .control-label {
+  width: 80px;
+}
+/* line 70, ../../app/assets/stylesheets/new_styles/_forms.scss */
+form.block-form fieldset .control-group .controls .field_with_errors input {
+  background-image: url(/assets/invalid_fat@2x.png);
+  background-repeat: no-repeat;
+  background-position: 197px;
+  background-size: 20px 20px;
+}
+/* line 81, ../../app/assets/stylesheets/new_styles/_forms.scss */
+form.block-form input[type=submit] {
+  text-align: center;
+  display: block;
+  width: 100%;
+  padding: 8px 0;
+}
+
+/* buttons to be extracted? */
+/* line 91, ../../app/assets/stylesheets/new_styles/_forms.scss */
+.new-btn {
+  -webkit-transition: box-shadow 0.1s linear;
+  -moz-transition: box-shadow 0.1s linear;
+  -o-transition: box-shadow 0.1s linear;
+  transition: box-shadow 0.1s linear;
+  -moz-border-radius: 5px 5px 5px 5px;
+  -webkit-border-radius: 5px 5px 5px 5px;
+  border-radius: 5px 5px 5px 5px;
+  background-image: #f4f4f4;
+  filter: progid:DXImageTransform.Microsoft.gradient(GradientType=0, startColorstr=$from, endColorstr=$to);
+  -ms-filter: "progid:DXImageTransform.Microsoft.gradient (GradientType=0, startColorstr=white, endColorstr=#e9e9e9)";
+  background-image: -webkit-gradient(linear, 0% 0%, 0% 100%, from(white), to(#e9e9e9));
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#e9e9e9` should only be used in variable declarations; they should be referred to via variable everywhere else.